mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -534,8 +534,10 @@ object MLUtils extends Logging {
       norm2: Double,
       precision: Double = 1e-6): Double = {
     val n = v1.size
-    require(v2.size == n)
-    require(norm1 >= 0.0 && norm2 >= 0.0)
+    require(v2.size == n,
+      s"Both vectors should have same length, found v1 is $n while v2 is ${v2.size}")
+    require(norm1 >= 0.0 && norm2 >= 0.0,
+      s"Both norms should be greater or equal to 0.0, found norm1=${norm1}, norm2=${norm2}")
```

Comment:
super nit: `${norm1}` -> `$norm1` and `${norm2}` -> `$norm2`?

Reply:
ok I will remove the `{` and `}`